cli/collection_update.go

```diff
@@ -48,38 +48,14 @@ Example: update private docIDs, with identity:
 			}
 
 			switch {
-			case len(argDocIDs) == 1 && updater != "":
-				docID, err := client.NewDocIDFromString(argDocIDs[0])
-				if err != nil {
-					return err
-				}
-				res, err := col.UpdateWithDocID(cmd.Context(), docID, updater)
-				if err != nil {
-					return err
-				}
-				return writeJSON(cmd, res)
-			case len(argDocIDs) > 1 && updater != "":
-				docIDs := make([]client.DocID, len(argDocIDs))
-				for i, v := range argDocIDs {
-					docID, err := client.NewDocIDFromString(v)
-					if err != nil {
-						return err
-					}
-					docIDs[i] = docID
-				}
-				res, err := col.UpdateWithDocIDs(cmd.Context(), docIDs, updater)
-				if err != nil {
-					return err
-				}
-				return writeJSON(cmd, res)
-			case filter != "" && updater != "":
+			case len(filter) > 0 && len(updater) > 0:
 				res, err := col.UpdateWithFilter(cmd.Context(), filter, updater)
 				if err != nil {
 					return err
 				}
 				return writeJSON(cmd, res)
-			case len(argDocIDs) == 1 && len(args) == 1:
-				docID, err := client.NewDocIDFromString(argDocIDs[0])
+			case len(argDocID) > 0 && len(args) == 1:
```

Comment:
todo: The use of `len(argDocID) > 0` is misleading here as it suggests that `argDocID` could be an array of `docID`s but then used as a string bellow. Please change this to `argDocID != ""`. Same thing in the `collection_delete.go` file :)